### PR TITLE
[WIP] Add Mobile Web Video Playback primer article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This a PR so that reviewers and others can have a look during work in progress.

Preview at https://pr-4299-dot-web-central.appspot.com/web/fundamentals/getting-started/primers/mobile-web-video-playback

### CODE REVIEW

**Comment by jpmedley** [comment]
@beaufortfrancois Are you about read for a copy edit?

**Comment by beaufortfrancois** [comment]
@jpmedley It isn't yet. I'll ping you for sure when it's ready. Thanks for asking!

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md [hunk 1]
```diff
@@ -22,54 +22,66 @@ controls, fullscreen, and background playback.
...
+    </figcaption>
+  </figure>
+</div>
 
-Video controls will contain as we implement them the play/pause button, the
+As you can see, the HTML layout we're going to use for our media player is
+pretty simple: a `<div>` root element contains a `<video>` media element and a
+`<div>` child element dedicated to video controls.
+
+Video controls will feature as we implement them a play/pause button, a
```
Comment (jpmedley) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: I don't understand this line. What will the video controls feature?

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md [hunk 2]
```diff
@@ -22,54 +22,66 @@ controls, fullscreen, and background playback.
...
 
 ### Read video metadata
 
-First, let's wait for the video metadata to be loaded to show the video
-duration and the current time. The `secondsToTimeCode` function is a custom utility
-function that converts a number of seconds to a string in "hh:mm:ss" format
-which is better suited in our case.
+First, let's wait for the video metadata to be loaded to set the video
+duration, the current time and initialize the progress bar. Note that the
+`secondsToTimeCode` function is a custom utility function I've written that
```
Comment (jpmedley) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: Function names should always include the (). Please check throughout.

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md [hunk 3]
```diff
@@ -93,10 +105,13 @@ playback state.
       }
     });
 
-Rather than adjusting our video controls in the `click` event listener, we'll
-use the fired `play` and `pause` video events. When video starts playing, we'll
-change button state to "pause", make sure time tracking is updated continuously
-and hide video controls. When video pauses, we'll simply change button state to
+Note: I call `event.stopPropagation()` to prevent parent handlers (eg.  video
```
Comment (jpmedley) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: Change 'eg.' to 'e.g.'.

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md [hunk 4]
```diff
@@ -117,17 +132,16 @@ and hide video controls. When video pauses, we'll simply change button state to
...
-only when needed and as efficiently as possible.
+Note: I use `requestAnimationFrame` to make sure time tracking is updated as
+efficiently as possible.
 
-When video ends, we'll simply change button state to "play" and show video
-controls for now as well. Note that we could also chose to load automatically
-the next video in the context of a playlist if user has enabled an "AutoPlay"
-feature.
+When video ends, we simply change button state to "play" and show video
+controls for now as well. Note that we could also choose to load automatically
```
Comment (jpmedley) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: As well as what? Should probably remove 'as well'.

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md (hunk 3 above)
Comment (beaufortfrancois) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: Done.

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md (hunk 4 above)
Comment (beaufortfrancois) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: Done.

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md (hunk 2 above)
Comment (beaufortfrancois) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: Done.

File: src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md (hunk 1 above)
Comment (beaufortfrancois) [inline src/content/en/fundamentals/getting-started/primers/mobile-web-video-playback.md]: Fixed. I've used 'cover' instead.

**Comment by beaufortfrancois** [comment]
It is now tracked at https://github.com/google/WebFundamentals/pull/4367

**Pull request closed without merging**